packages/mandelbrot/src/theme.js

```diff
@@ -6,6 +6,53 @@ const Theme = require('@frctl/web').Theme;
 const packageJSON = require('../package.json');
 
 module.exports = function (options) {
+    const labels = _.defaultsDeep(_.clone((options || {}).labels || {}), {
```

Comment:
Define `options = _.clone(options || {})` on the first line to avoid cloning it twice, once for the `labels` and another for the `config`? Also shouldn't it be `cloneDeep`? Or is cloning required at all? I’m confused 😆

Reply:
Actually I don’t think I understand why you extracted labels. I have the feeling it does exactly the same as before. What am I missing?

Reply:
Ah yes, I changed where I assigned the default information entry at the last minute, so now this change is not necessary at all. Good catch 👍